docs/tutorial/part-five/index.md

```diff
@@ -75,47 +75,55 @@ module.exports = {
 }
 ```
 
-Save that and restart the gatsby development server. Then open up GraphiQL again.
+Simpan itu dan mulai ulang server pengembangan gatsby. Kemudian buka kembali GraphiQL.
 
-In the explorer pane, you'll see `allFile` and `file` available as selections:
+Di panel penjelajah, Anda akan melihat `allFile` dan `file` tersedia sebagai pilihan:
 
 ![graphiql-filesystem](graphiql-filesystem.png)
 
-Click the `allFile` dropdown. Position your cursor after `allFile` in the query area, and then type <kbd>Ctrl + Enter</kbd>. This will pre-fill a query for the `id` of each file. Press "Play" to run the query:
+Klik dropdown `allFile`. Posisikan kursor Anda setelah `allFile` di area permintaan, dan lalu ketik <kbd>Ctrl + Enter</kbd>. Ini akan mengisi `id` untuk setiap file terlebih dahulu. Tekan "Mainkan" untuk menjalankan kueri:
 
 ![filesystem-query](filesystem-query.png)
 
-In the Explorer pane, the `id` field has automatically been selected. Make selections for more fields by checking the field's corresponding checkbox. Press "Play" to run the query again, with the new fields:
+Di panel Exploere, bidang `id` telah dipilih secara otomatis. Tentukan pilihan untuk lebih banyak bidang dengan mencentang kotak yang sesuai pada bidang tersebut. Tekan "Mainkan" untuk menjalankan kueri lagi, dengan bidang baru:
 
-![filesystem-explorer-options](filesystem-explorer-options.png)
+![pilihan-pada-filesystem-explorer](filesystem-explorer-options.png)
 
-Alternatively, you can add fields by using the autocomplete shortcut (<kbd>Ctrl + Space</kbd>). This will show queryable fields on the `File` nodes.
+Alternatifnya, Anda dapat menambahkan bidang dengan menggunakan pintasan _autocomplete_ (<kbd>Ctrl + Space</kbd>). Ini akan menampilkan bindang yang dapat diminta pada node `File`.
 
 ![filesystem-autocomplete](filesystem-autocomplete.png)
 
-Try adding a number of fields to your query, pressing <kbd>Ctrl + Enter</kbd>
-each time to re-run the query. You'll see the updated query results:
+Coba tambahkan sejumlah bidang ke kueri Anda, dengan menekan <kbd>Ctrl + Enter</kbd>
+setiap kali menjalankan kueri kembali. Anda akan melihat hasil _query_ yang diperbarui:
 
 ![allfile-query](allfile-query.png)
 
-The result is an array of `File` "nodes" (node is a fancy name for an object in a
-"graph"). Each `File` node object has the fields you queried for.
+Hasilnya adalah array dari `File` "nodes" (_node_ adalah nama mewah untuk objek di "_graph_"). Setiap objek `File` _node_ memiliki bidang yang Anda minta.
 
-## Build a page with a GraphQL query
+## Membuat halaman dengan GraphQL _query_
+
+Membangun halaman baru dengna Gatsby sering dimulai dengan GraphiQL. Anda pertama-tama membuat sketsa
+kueri data dengan bermain di GraphiQL lalu salin ini ke halaman komponen React
+untuk mulai membangun UI.
+
+Ayo Coba ini.
 
 Building new pages with Gatsby often starts in GraphiQL. You first sketch out
```

Comment:
I think there some miss line in here, in line 104 to 109 is the translation of this paragraph.

Reply:
Yeah, sorry i forget to delete the original paragraph